highfive, libsonata: conflicts with 'highfive@2.5,2.6'.

1uc opened the pull request.

1uc committed:
libsonata: conflicts with `^highfive@2.5:2.6`.

matz-e commented on bluebrain/repo-bluebrain/packages/libsonata/package.py:
```diff
@@ -45,6 +45,8 @@ class Libsonata(CMakePackage):
     # version penalty from `nlohmann-js` then :(
     depends_on("nlohmann-json@:3.9.1")
 
+    conflicts("highfive@2.5,2.6")
```
```suggestion
    conflicts("^highfive@2.5:2.6")
```
?

1uc committed:
Release highfive `2.7.0`.

1uc committed:
Pre-release v2.7.1

1uc committed:
Second pre-release of `highfive@2.7.1`.

1uc committed:
Page buffer patch for `highfive@2.7.1`.

Since libsonata has a conflict with 2.6.2, we can't
build morpho-kit without the pagebuffer patch for 2.7.1.

1uc committed:
Whitespace & typo.

1uc committed:
Third pre-release.

1uc committed:
Re-run title check.

1uc committed:
The official highfive@2.7.1
